src/main/java/at/hannibal2/skyhanni/features/mining/crystalhollows/CrystalHollowsLobbyAgeWarning.kt

```diff
@@ -0,0 +1,100 @@
+
+    private fun processPlayerCount() {
+        for (line in TabListData.getTabList()) {
+            playerCountTabListPattern.matchMatcher(line) {
+                val playerCount = group("playerCount").formatNumber()
+                if (oldPlayerCount == playerCount) return
+                oldPlayerCount = playerCount
+                if (config.playerCountReminders) LorenzUtils.chat("§a$playerCount players are currently in this Crystal Hollows lobby.")
+                if (!lobbyIsPastAgeThreshold) return
+                if (playerCount in 1..minPlayers) LorenzUtils.chat("§cThere are $playerCount players remaining in this Crystal Hollows lobby. §4§lIt will shut down very soon.")
+                else LorenzUtils.chat("There are $playerCount players remaining in this Crystal Hollows lobby. §cIt will shut down when there are fewer than $minPlayers people left, or when this lobby reaches Day $maxLobbyAgeMCDays.")
+            }
+        }
+    }
+
+    private fun processLobbyAge() {
+        val lobbyAge = getLobbyAgeInMinecraftDays()
+        if (oldLobbyAge == lobbyAge) return
+        oldLobbyAge = lobbyAge
+        if (config.lobbyAgeReminders && !lobbyIsPastAgeThreshold) LorenzUtils.chat("§aThis Crystal Hollows lobby is currently at Day $lobbyAge, ${StringUtils.optionalPlural(number = abs(minLobbyAgeMCDays - lobbyAge).toInt(),"day", plural = "days")} away from Day $minLobbyAgeMCDays.")
+        if (lobbyAge in minLobbyAgeMCDays..< maxLobbyAgeMCDays && !lobbyIsPastAgeThreshold) {
+            lobbyIsPastAgeThreshold = true
+            LorenzUtils.chat("This Crystal Hollows lobby has reached Day $minLobbyAgeMCDays. It no longer accepts new players, §cand will shut down on Day $maxLobbyAgeMCDays or when there are fewer than $minPlayers people left.")
+        } else if (lobbyAge >= maxLobbyAgeMCDays) LorenzUtils.chat("§cThis Crystal Hollows lobby has reached Day $maxLobbyAgeMCDays. §4§lIt will shut down very soon.")
+    }
+
+    @SubscribeEvent
+    fun onTick(event: LorenzTickEvent) {
+        if (!isEnabled()) return
+        if (!event.repeatSeconds(1)) return
+        processLobbyAge()
+    }
+
+    @SubscribeEvent
+    fun onTabListUpdate(event: TabListUpdateEvent) {
+        if (!isEnabled()) return
+        processPlayerCount()
+    }
+
+    @SubscribeEvent
+    fun onIslandChange(event: IslandChangeEvent) {
+        if (event.newIsland != THE_CRYSTAL_HOLLOWS) {
+            lobbyIsPastAgeThreshold = false
+            oldPlayerCount = 0L
+            oldLobbyAge = 0L
+        }
+    }
+    
+    @SubscribeEvent
+    fun onRepoReload(event: RepositoryReloadEvent) {
+        val data = event.getConstant<CrystalHollowsLobbyAgeWarningJson>("CrystalHollowsLobbyAgeWarning")
+        minPlayers = data.minPlayers
+        minLobbyAgeMCDays = data.minLobbyAgeMCDays
+        maxLobbyAgeMCDays = data.maxLobbyAgeMCDays
```

Comment:
```suggestion
        minLobbyAge = data.minLobbyAge
        maxLobbyAge = data.maxLobbyAge
```
Dont need to specify that